internal/app/s3manager/bucket_view.go

```diff
@@ -45,7 +45,11 @@ func HandleBucketView(s3 S3, tmplDir string) http.HandlerFunc {
 
 		l := filepath.Join(tmplDir, "layout.html.tmpl")
 		p := filepath.Join(tmplDir, "bucket.html.tmpl")
-		t, err := template.ParseFiles(l, p)
+		t, err := template.New("").Funcs(template.FuncMap{
```

Comment:
It is not necessary to use the `Funcs` pattern. Rather, we should just add the `basePath` to the respective `pageDate` struct as a property.